### ISSUE TITLE
"Enter," Chat Window Shortcut Conflicts with Accessibility SUpport

### ISSUE THREAD

**Issue by BlindGuyNW** [MegaMek/megamek#2625]
### Environment
What version of MegaMek does your issue apply to?  Megamek version 0.47.16, stand-alone.
What operating system are you using?  Mac.
What java version are you using? Java/OpenJDK version 1.8.0_282.

### Description
The default shortcut key used to open the chat window, "Enter," conflicts with the Accessibility window used to control the game with a screen reader. 

Steps:
1. Launch Megamek.
2. Open a scenario.
3. Open the accessibility window from the View menu.
4. Try to type a command, such as #help or #tile. Press Enter to submit.

Actual results:
The focus jumps from the Accessibility window, and the command is not submitted. The chat window presumably opens.

Expected Results:
Focus should remain in the command box. The command should be submitted. The chat window should remain closed.

Possible Resolutions:
Change the default chat key shortcut to something less frequently used, such as the grave accent. Long-term, perhaps work on making the chat window accessible and integrating the separate "accessibility window," functionality into it.

### Files
NA

**Comment by Windchild292** [comment]
Ideas for alternative short term fixes:
The accessibility window enter could be made customizable, or swap chat to a fall back (like grave) when the window is enabled?

A further query, would making the accessibility window scrollable help?

**Comment by BlindGuyNW** [comment]
@Windchild292. I like the idea of making the shortcut default to something less frequent if the window is open. The shortcut dialogue is actually kind of a nightmare for accessibility in itself, so if we didn’t have to customize it that would be ideal. :-) As far as scrolling is concerned, that probably is something that would help things be a little better visually, but it doesn’t seem to bother assistive technology any. Probably doesn’t hurt though.

**Comment by Windchild292** [comment]
The window is not capturing any keys, and as far as I can tell it has never properly worked (tested on 0.47.2).

**Comment by BlindGuyNW** [comment]
Huh.

I remember when I originally worked on this, my working copy for some reason had the chat shortcut key disabled. That is, I could use Return to submit commands as expected. When I sent the code for inclusion in master, the shortcut key somehow became an issue. Not sure what happened there.

Is there another issue you’re having beyond the chat window stealing focus? It’s hard for me to tell what’s going on, it’s entirely possible there are other major problems with it.

**Comment by Windchild292** [comment]
All shortcut keys take affect when using the window, so it is very feasible that one using it will accidentally make changes to the game while typing in the dialog.

**Comment by BlindGuyNW** [comment]
Ahh. That's fair. Evens shortcuts for unit movement and such? That's not good. I never noticed it. :/ Is there a way to disable the shortcut processing when the dialog is in focus?
